### PR TITLE
Jobs to test COO

### PR DESCRIPTION
**Pull request by lihongyan1**
Weekly jobs to run with QE index image
Yearly jobs to run with stage image,  trigger job with curl command when need Yearly job to run with productoin image, trigger job with curl command when need
[detail doc](https://docs.google.com/document/d/1U2LKqRSI3hWIQXAELcxT9eczKIQODrQ889evLioteM0/edit)

### CODE REVIEW

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp415-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp414-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp413-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp412-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp411-coo-stage

**Comment by lihongyan1** [comment]
periodic-ci-rhobs-observability-operator-main-ocp414-coo-qeindex

**Comment by lihongyan1** [comment]
periodic-ci-rhobs-observability-operator-main-ocp413-coo-qeindex

**Comment by lihongyan1** [comment]
periodic-ci-rhobs-observability-operator-main-ocp412-coo-qeindex

**Comment by lihongyan1** [comment]
periodic-ci-rhobs-observability-operator-main-ocp411-coo-qeindex

**Comment by lihongyan1** [comment]
/pj-rehearse  periodic-ci-rhobs-observability-operator-main-ocp-coo-interop-production

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp411-coo-qeindex

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp411-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp412-coo-stage
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp413-coo-stage
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp414-coo-stage
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp415-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp413-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp414-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp415-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp411-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp412-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp413-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp414-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp415-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp412-coo-qeindex

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp413-coo-qeindex

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp414-coo-qeindex

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp415-coo-qeindex

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp-coo-interop-production

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp411-coo-qeindex

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp412-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp412-coo-qeindex

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp411-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp413-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp413-coo-qeindex

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp414-coo-qeindex

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp415-coo-qeindex

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp414-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp415-coo-stage

**Comment by lihongyan1** [comment]
/pr-rehearse periodic-ci-rhobs-observability-operator-main-ocp-coo-interop-production

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp411-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp412-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp413-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp414-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp415-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp-coo-interop-production

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp415-coo-qeindex

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp414-coo-qeindex

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp413-coo-qeindex

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp412-coo-qeindex

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp411-coo-qeindex

**Comment by lihongyan1** [comment]
/do-not-merge/work-in-progress

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp411-coo-qeindex

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp411-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp412-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp412-coo-qeindex

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp413-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp413-coo-qeindex

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp414-coo-qeindex

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp414-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp415-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp415-coo-qeindex

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp-coo-interop-production

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp411-coo-qeindex

**Comment by lihongyan1** [comment]
/pj-rehearse  periodic-ci-rhobs-observability-operator-main-ocp411-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp-coo-interop-production

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp415-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp414-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp413-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp412-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp415-coo-qeindex

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp414-coo-qeindex

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp413-coo-qeindex

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp412-coo-qeindex

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp411-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp412-coo-qeindex

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp412-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp413-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp413-coo-qeindex

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp414-coo-qeindex

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp414-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp415-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp415-coo-qeindex

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp412-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp413-coo-qeindex

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp413-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp414-coo-qeindex

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp414-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp415-coo-qeindex

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp415-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp412-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp413-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp412-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp412-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp413-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp412-coo-stage

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp412-coo-stage

File: ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main__ocp-coo-interop.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+resources:
+  '*':
+    limits:
+      memory: 4G
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: production
+  cron: 0 6 1 7 *
```
Comment (danielmellado) [inline ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main__ocp-coo-interop.yaml:30]: While the config LGTM, I'm wondering if a *yearly* trigger wouldn't be just too little. Is there any motivation for not triggering this a bit more often?

File: ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main__ocp411-coo.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      OPERATORS: |
+        [
+            {"name": "cluster-observability-operator", "source": "qe-app-registry", "channel": "development", "install_namespace": "openshift-coo", "target_namespaces": "", "operator_group":"openshift-coo-og"}
+        ]
+    test:
+    - ref: install-operators
+    - ref: rhobs-obo-test-harness
+    workflow: cucushift-installer-rehearse-gcp-upi
+- as: stage
+  cron: 0 6 1 12 *
```
Comment (danielmellado) [inline ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main__ocp411-coo.yaml:47]: same here, I'm also wondering if there's any reason behind the specific dates or if they were just random ones.

File: ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main__ocp412-coo.yaml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+      OPERATORS: |
+        [
+            {"name": "cluster-observability-operator", "source": "qe-app-registry", "channel": "development", "install_namespace": "openshift-coo-production", "target_namespaces": "", "operator_group":"openshift-coo-production-og"}
+        ]
+    test:
+    - ref: install-operators
+    - ref: rhobs-obo-test-harness
+    workflow: cucushift-installer-rehearse-aws-ipi-ovn-ipsec
+- as: stage
+  cron: 0 6 1 11 *
```
Comment (danielmellado) [inline ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main__ocp412-coo.yaml:44]: same

File: ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main__ocp413-coo.yaml [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+      OPERATORS: |
+        [
+            {"name": "cluster-observability-operator", "source": "qe-app-registry", "channel": "development", "install_namespace": "openshift-coo-production", "target_namespaces": "", "operator_group":"openshift-coo-production-og"}
+        ]
+    test:
+    - ref: install-operators
+    - ref: rhobs-obo-test-harness
+    workflow: cucushift-installer-rehearse-gcp-upi-xpn
+- as: stage
+  cron: 0 6 1 10 *
```
Comment (danielmellado) [inline ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main__ocp413-coo.yaml:47]: same

File: ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main__ocp414-coo.yaml [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+      OPERATORS: |
+        [
+            {"name": "cluster-observability-operator", "source": "qe-app-registry", "channel": "development", "install_namespace": "openshift-coo", "target_namespaces": "", "operator_group":"openshift-coo-og"}
+        ]
+    test:
+    - ref: install-operators
+    - ref: rhobs-obo-test-harness
+    workflow: cucushift-installer-rehearse-azure-ipi-sdn
+- as: stage
+  cron: 0 6 1 9 *
```
Comment (danielmellado) [inline ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main__ocp414-coo.yaml:44]: same

File: ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main__ocp415-coo.yaml [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+      OPERATORS: |
+        [
+            {"name": "cluster-observability-operator", "source": "qe-app-registry", "channel": "development", "install_namespace": "openshift-coo", "target_namespaces": "", "operator_group":"openshift-coo-og"}
+        ]
+    test:
+    - ref: install-operators
+    - ref: rhobs-obo-test-harness
+    workflow: cucushift-installer-rehearse-aws-ipi-shared-vpc-phz-sts
+- as: stage
+  cron: 0 6 1 8 *
```
Comment (danielmellado) [inline ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main__ocp415-coo.yaml:44]: same

**Review by danielmellado — COMMENTED** [review]
The config looks overall good to me, I just have a few questions about the triggers, thanks!

File: ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main__ocp-coo-interop.yaml (hunk 1 above)
Comment (lihongyan1) [reply ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main__ocp-coo-interop.yaml:30]: I set it as yearly trigger for I plan to trigger the job with command when an errata is delivered

File: ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main__ocp411-coo.yaml (hunk 2 above)
Comment (lihongyan1) [reply ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main__ocp411-coo.yaml:47]: It's a random yearly date, for I plan to trigger the job with command when an errata builds are push to stage

File: ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main__ocp412-coo.yaml (hunk 3 above)
Comment (lihongyan1) [reply ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main__ocp412-coo.yaml:44]: It's a random yearly date, for I plan to trigger the job with command when errata builds are push to stage

File: ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main__ocp413-coo.yaml (hunk 4 above)
Comment (lihongyan1) [reply ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main__ocp413-coo.yaml:47]: It's a random yearly date, for I plan to trigger the job with command when errata builds are push to stage

File: ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main__ocp414-coo.yaml (hunk 5 above)
Comment (lihongyan1) [reply ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main__ocp414-coo.yaml:44]: It's a random yearly date, for I plan to trigger the job with command when errata builds are push to stage

File: ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main__ocp415-coo.yaml (hunk 6 above)
Comment (lihongyan1) [reply ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main__ocp415-coo.yaml:44]: It's a random yearly date, for I plan to trigger the job with command when errata builds are push to stage

File: ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main__ocp-coo-interop.yaml (hunk 1 above)
Comment (danielmellado) [reply ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main__ocp-coo-interop.yaml:30]: ack, thanks for the heads up! /lgtm then

**Pull request merged**